sweep/tx_input_set.go

```diff
@@ -151,6 +151,15 @@ func (t *txInputSet) enoughInput() bool {
 		return false
 	}
 
+	// NOTE: if we decide to have no change output, we can return true here
+	// as the fee can always be covered - we don't necessarily need a
+	// non-dust change output, as aggregating UTXOs to keep the set small
+	// is also important.
+	//
+	// TODO(yy): make the weightEstimate take account of the outputs to get
```

Comment:
Do you mean the required outputs here?

Reply:
nah I understood it wrong here, removed the comments🤦🏻